lib/actions/sessions.js

```diff
@@ -167,3 +171,59 @@ export function exitSessionBrowser(uid) {
     uid
   };
 }
+
+export function resetSessionFontSize() {
+  return (dispatch, getState) => {
+    dispatch({
+      type: SESSION_FONT_SIZE_RESET,
+      effect() {
+        const {sessions} = getState();
+        const uid = sessions.activeUid;
+        const value = 0;
+        dispatch({
+          type: SESSION_FONT_SIZE_SET,
+          uid,
+          value
+        });
+      }
+    });
+  };
+}
+
+export function increaseSessionFontSize() {
+  return (dispatch, getState) => {
+    dispatch({
+      type: SESSION_FONT_SIZE_INCR,
+      effect() {
+        const {sessions} = getState();
+        const uid = sessions.activeUid;
+        const old = sessions.sessions[uid].fontSizeOverride;
+        const value = old + 1;
+        dispatch({
+          type: SESSION_FONT_SIZE_SET,
+          uid,
+          value
+        });
+      }
+    });
+  };
+}
```

Comment:
I think you don't need using  `effect` mechanism and to add `SESSION_FONT_SIZE_INCR` and `SESSION_FONT_SIZE_DECR` action types.

Something like:

Reply:
@chabou I already removed the effect mechanism of the session font size increase and decrease. Do you want me to do it for the session font size reset too?